tests/behat/features/bootstrap/WebFeatureContext.php

```diff
@@ -1700,49 +1701,34 @@ public function iDownload($arg1)
     $this->getClient()->request('GET', $arg1);
   }
 
-  /**
-   * @Then /^I should receive a "([^"]*)" file$/
-   * @param $extension
-   */
-  public function iShouldReceiveAFile($extension)
-  {
-    $content_type = $this->getClient()->getResponse()->headers->get('Content-Type');
-    Assert::assertEquals('image/' . $extension, $content_type);
-  }
 
   /**
    * @Then /^I should receive a file named "([^"]*)"$/
    * @param $name
    */
   public function iShouldReceiveAFileNamed($name)
   {
-    $content_disposition = $this->getClient()->getResponse()->headers->get('Content-Disposition');
-    Assert::assertEquals('attachment; filename="' . $name . '"', $content_disposition);
-  }
+    $received = false;
+    $file_path = self::BROWSER_DOWNLOAD_DIR . '/' . $name;
 
-  /**
-   * @Given /^the response code should be "([^"]*)"$/
-   * @param $code
-   */
-  public function theResponseCodeShouldBe($code)
```

Comment:
why have you removed this snippet?
Now the web suite has 1 undefined step:

tests/behat/features/web/project_download.feature line 19.

Reply:
This happened accidentally while rebasing and merging. Already fixed an force pushed.